bin/build.sh

```diff
@@ -48,8 +48,8 @@ echo -e "${grey}  Packaging waffle.io-migrations...${reset}"
 docker save --output="waffleio-takeout/images/waffle.io-migrations.tar" quay.io/waffleio/waffle.io-migrations
 echo -e "${grey}  Packaging waffle.io-rally-integration...${reset}"
 docker save --output="waffleio-takeout/images/waffle.io-rally-integration.tar" quay.io/waffleio/waffle.io-rally-integration
-echo -e "${grey}  Packaging nginx-proxy...${reset}"
-docker save --output="waffleio-takeout/images/nginx-proxy.tar" jwilder/nginx-proxy
+echo -e "${grey}  Packaging takeout-nginx...${reset}"
```

Comment:
All our other images are called `waffle-*`, might be weird for admins to have to know both prefixes.